db-client-java/src/main/java/com/eventstore/dbclient/EventStoreDBClient.java

```diff
@@ -145,7 +170,7 @@ public Publisher<ResolvedEvent> readAllReactive(ReadAllOptions options) {
         return this.readAllReactive(Long.MAX_VALUE, options);
     }
 
-    public Publisher<ResolvedEvent> readAllReactive(long maxCount, ReadAllOptions options) {
+    public Publisher<ResolvedEventWithContext> readAllReactiveWithContext(long maxCount, ReadAllOptions options) {
```

Comment:
**CONCERN:** @YoEight, as you derived `ResolvedEventWithContext` from `ResolvedEvent`, then this method seems redundant. I think that it's okay to just have `readAllReactive` that returns `ResolvedEventWithContext`. Am I missing some important point here?

Reply:
Done.